code/datums/uplink_item.dm

```diff
@@ -549,6 +557,13 @@ var/list/uplink_items = list()
 	cost = 1
 	surplus = 50
 
+/datum/uplink_item/stealthy_weapons/traitor_virus_kit
+	name = "Virus Kit"
+	desc = "An active fungal pathogen in a sterile, compact box. Comes with one Bio Virus Antidote Kit (BVAK) autoinjector for rapid application on upto two targets each, a syringe, and a bottle containing the BVAK solution."
+	item = /obj/item/weapon/storage/box/syndie_kit/tuberculosiskit
+	cost = 9
+	surplus = 50
```

Comment:
That's the virus kit, I think the virus kit shouldn't be restricted to operator only.